pages/documents/ConversationalAI/ThirdPartyBots/PublicApiV1.md

```diff
@@ -41,22 +41,37 @@ To identify the Third-Party Bots API endpoint user, first get domain information
 
 #### Step 2. Get Bearer Token
 
-To get the bearer token you must perform a `login` request to Third-Party Bots API Domain. To perform login request you will need a valid user created via LivePerson User Management UI. User must be `Enabled`, have minimum `Agent` role and **Login Method** set to `Password`.
+To use our Public API you must perform a login request to Third-Party Bots API domain which will in response send you a Third-Party Bots bearer token. There are two ways in which you can perform the login.
+
+<ul>
+  <li>Bearer Token via Username/Password</li>
+  <li>Bearer Token via API (OAuth)</li>
+</ul>
+
+To perform login request you will need a valid Bot user created via LivePerson User Management UI. User must be `Enabled`, have minimum `Agent` role and have a **Login Method** selected either to `Password` or `API Key` as shown in Figure 2.1.
```

Comment:
To perform login requests you will need a valid Bot user created via LivePerson User Management UI. User must be `Enabled`, have minimum `Agent` role and have a **Login Method** selected either to `Password` or `API Key` as shown in Figure 2.1.